src/Nethermind/Nethermind.Consensus/Transactions/BaseFeeTxFilter.cs

```diff
@@ -0,0 +1,55 @@
+﻿//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+// 
+
+using Nethermind.Core;
+using Nethermind.Core.Specs;
+using Nethermind.Logging;
+
+namespace Nethermind.Consensus.Transactions
+{
+    public class BaseFeeTxFilter : ITxFilter
```

Comment:
Add comments to classes

Reply:
Hmm - no problem, but could we discuss it on any occasion? I don't want to create comments like BaseFeeTxFilter - the class filters transactions based on base fee.